src/native/libs/System.Globalization.Native/pal_collation.c

```diff
@@ -334,11 +334,11 @@ static UCollator* CloneCollatorWithOptions(const UCollator* pCollator, int32_t o
         }
 #else // !defined(STATIC_ICU)
 
-#if HAVE_UCOL_CLONE
+#if HAVE_UCOL_CLONE || (defined(U_ICU_VERSION_MAJOR_NUM) && U_ICU_VERSION_MAJOR_NUM >= 72)
```

Comment:
Why is it not enough to set `HAVE_UCOL_CLONE` in `configure.cmake`?

Reply:
It is, it looks like I did not check it properly. Thanks.